jekyll/_cci2/contexts.md

```diff
@@ -41,11 +41,13 @@ If you find you need to rename an org or repo that you have previously hooked up
 
 ## Creating and Using a Context
 
-1. Using the new version of the CircleCI application, navigate to the Organization Settings page by clicking on the link in the sidebar. **Note:** Any organization member can create a context, but only organization administrators can restrict it with a security group.
+1. Using the new version of the CircleCI application, navigate to the Organization Settings page by clicking on the link in the sidebar. 
+
+**Note:** Organization members can create a context, but only organization administrators can restrict it with a security group. The one exception to this case is BitBucket organizations, which require a user to have the `create repositories` workkspace permission, regardless of their other permissions on the workspace or the repositories it contains.
```

Comment:
```suggestion
    **Note:** Organization members can create a context, but only organization administrators can restrict it with a security group. The one exception to this case is BitBucket organizations, which require a user to have the `create repositories` workkspace permission, regardless of their other permissions on the workspace or the repositories it contains.
```

Reply:
Not sure this fixes it, would have to check locally, but the para needs indenting to keep the numbered steps flowing correctly.

Reply:
There is a small typo:
**_workkspace_** instead of **_workspace_**, just after `create repositories`

Reply:
fixed both issues 👍